src/autowrap.lisp

```diff
@@ -7,7 +7,8 @@
   :exclude-sources ("/usr/local/lib/clang/([^/]*)/include/(?!stddef.h)"
                     "/usr/include/"
                     "/usr/include/arm-linux-gnueabihf"
-                    "/usr/local/include/SDL2")
+                    "/usr/include/SDL2"
```

Comment:
You changed `"/usr/local/include/SDL2"` unconditionally to `"/usr/include/SDL2"`.  If `Darwin` needs to exclude `"/usr/include/SDL2"`, should it not be `#+darwin "/usr/include/SDL2"` in addition to `"/usr/local/include/SDL2"` to avoid influencing other platforms?

Reply:
On my macos-14 arm64 after `brew install sdl2`, it's:

/opt/homebrew/include/SDL2/

I don't know if this would help any.

Reply:
I believe this change accidentally made its way into the PR, but it was not required.  I have reverted it now.